docs/getting-started.md

```diff
@@ -244,6 +244,15 @@ available hardware.
 
 {% include /style/begin-section.html %}
 
+## Step 5.1: Update `apt`
+
+Currently, the image file downloaded from step 1 was built in 2020,
+some of its built-in packages are dated, and `apt` does not work out of the box.
+
+To fix `apt` in general, `/etc/apt/sources.list` needs to be changed to match
+https://github.com/ev3dev/docker-library/blob/master/ev3dev-stretch/layers/debian/etc/apt/sources.list
```

Comment:
Could we suggest how to do this for less experienced users? For example `sudo nano /etc/apt/sources.list` may be the simplest way.